src/components/values/OurValuesCard.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+
+export enum OurValuesIconType {
+	LEVELS = "levels",
+	IDEA = "idea",
+	THUMBSUP = "thumbsup",
+}
```

Comment:
Itt talan egyszerubb, ha string literalt hasznalunk:

```
type OurValuesIconType = "levels" | "idea" | "thumbsup";
```

Ebben az esetben nem is kell exportalni kulon hogy hasznalhassuk, es a TS miatt ezeken kivuli erteken nem is enged.